src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -199,14 +199,15 @@ def remove_sync_policy_group(self, group_id: str, bucket_name=''):
     @Endpoint(method='PUT', path='/sync-flow')
     @EndpointDoc("Create or update the sync flow")
     @CreatePermission
+    # pylint: disable=W0102
     def create_sync_flow(self, flow_id: str, flow_type: str, group_id: str,
-                         source_zone: Optional[List[str]] = None,
-                         destination_zone: Optional[List[str]] = None,
-                         zones: Optional[List[str]] = None,
+                         source_zone: Optional[str] = None,
+                         destination_zone: Optional[str] = None,
+                         zones: Optional[Dict[str, Any]] = {'added': [], 'removed': []},
```

Comment:
why Any ? when in default we putting List ? is it possible to do :
```suggestion
                         zones: Optional[Dict[str, List]] = None,
                         if zones is None:
                         xones = {'added': [], 'removed': []}
```